packages/main/src/Popup.js

```diff
@@ -278,10 +278,17 @@ class Popup extends UI5Element {
 	}
 
 	_onfocusout(e) {
-		// relatedTarget is the element, which will get focus. If no such element exists, focus the root
+		// relatedTarget is the element, which will get focus. If no such element exists, focus the root.
 		if (!e.relatedTarget) {
 			this._root.tabIndex = -1;
```

Comment:
move this to just before the this._root.focus()

Reply:
Done